Gems/Atom/Feature/Common/Code/Include/Atom/Feature/Utils/FrameCaptureBus.h

```diff
@@ -86,13 +86,17 @@ namespace AZ
             : public EBusTraits
         {
         public:
-            virtual ~FrameCaptureNotifications() = default;
+            static const EBusHandlerPolicy HandlerPolicy = EBusHandlerPolicy::Multiple;
+            static const AZ::EBusAddressPolicy AddressPolicy = AZ::EBusAddressPolicy::ById;
+            using BusIdType = FrameCaptureId;
+
+            virtual ~FrameCaptureNotifications() {};
             
             //! Notify when a capture is finished, you may receive notifications for other captures than your own
-            //! @param frameCaptureId The frame capture id returned when the capture was triggered
+            //! @param frameCaptureId The frame capture id returned when the capture was triggered. This is returned here in case client code has triggered multiple captures
             //! @param result result code
             //! @param info The output file path or error information which depends on the result. 
-            virtual void OnCaptureFinished(uint32_t frameCaptureId, FrameCaptureResult result, const AZStd::string& info) = 0;
+            virtual void OnFrameCaptureFinished(FrameCaptureId frameCaptureId, FrameCaptureResult result, const AZStd::string& info) = 0;
```

Comment:
Since you have to connect by ID, you should not need to pass the ID into the function anymore. You can get the active bus ID from the handler from within the function call.